polymer.json

```diff
@@ -7,5 +7,14 @@
     "src/shop-cart.html",
     "src/shop-checkout.html",
     "src/lazy-resources.html"
+  ],
+  "sourceGlobs": [
+   "src/**/*",
+   "images/**/*",
+   "bower.json"
+  ],
+  "includeDependencies": [
+    "manifest.json",
+    "bower_components/webcomponentsjs/webcomponents-lite.min.js"
```

Comment:
@robdodson Thanks for the fix!

@robdodson and @FredKSchott, we also need to copy the `data` folder to the build.  Should we put that under `sourceGlobs`?